deepchecks/vision/checks/methodology/simple_feature_contribution.py

```diff
@@ -120,8 +116,8 @@ def update(self, context: Context, batch: Any, dataset_kind: DatasetKind):
 
             properties['target'] += classes
         else:
-            raise DeepchecksValueError(
-                f'Check {self.__class__.__name__} does not support task type {dataset.task_type}')
+            imgs = batch.images
+            properties['target'] += dataset.get_classes(batch.labels)
```

Comment:
maybe just use get_classes in all of them and be agnostic?

Reply:
but in object detection we do extra things on the label anyway (like cropping image and bboxes)

Reply:
Done